Sources/HaCTML/render.swift

```diff
@@ -0,0 +1,64 @@
+import HTMLString
```

Comment:
Worth considering this dependency again if and when we plan to make this open source

Reply:
A candidate replacement: https://github.com/IBM-Swift/swift-html-entities

Reply:
:+1: